app/modules/conversations/conversation/conversation_service.py

```
@@ -664,6 +665,25 @@ async def get_conversation_info(
             )
+
+            agent_id = conversation.agent_ids[0] if conversation.agent_ids else None
+
+            if agent_id:
+                system_agents = self.agent_service._system_agents(
+                    self.provider_service, 
+                    self.prompt_service,
+                    self.tool_service
+                )
+                
+                if agent_id in system_agents.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)